src/config/Configuration.tsx

```diff
@@ -15,6 +15,12 @@ export default class Configuration {
     { id: 'scpqa', name: 'SAP Cloud Platform QA', endpoint: Configuration.SCP_REST_ENDPOINT_QA }
   ];
 
+  public static readonly ENDPOINT_CLOUDS_DEV: EndpointCloud[] = [
+    ...Configuration.ENDPOINT_CLOUDS,
+    { id: 'localhost:8020', name: '127.0.0.1:8020', endpoint: 'http://127.0.0.1:8020' },
+    { id: 'android:localhost:8020', name: '10.0.2.2:8020', endpoint: 'http://10.0.2.2:8020' }
+  ];
```

Comment:
The setup you put in there is specific to your local development environment and can't be hardcoded. You need to introduce a configuration file that permit to define custom endpoints for development purpose.